paddle/operators/conv_cudnn_op.cu.cc

```diff
@@ -155,19 +200,34 @@ class CudnnConvGradOpKernel : public framework::OpKernel<T> {
     cudnnTensorDescriptor_t cudnn_input_grad_desc = nullptr;
```

Comment:
`cudnn_input_grad_desc ` and `cudnn_input_desc ` are the same, you can replace `cudnn_input_grad_desc ` with `cudnn_input_desc`.  Just like [this](https://github.com/PaddlePaddle/Paddle/blob/65c859db7aadfdaccb1a04afe788d66d0e4a8694/paddle/operators/pool_cudnn_op.cu.cc#L129).